npc/cities/aldebaran.txt

```diff
@@ -1098,28 +1098,28 @@ aldeba_in,79,161,6	script	Kafra#04	4_F_KAFRA3,{
 		.@list$ = "";
 		.@numitemchoices = 0;
 		for (.@i = 0; .@i < getarraysize(.@choices) - 3; .@i += 3) { // Skip the last entry as it's handled separately
-			.@list$ += .@choices[.@i] + "- " + getitemname(.@choices[.@i+1]) + " " + .@choices[.@i+2] + " ea:";
+			.@list$ +=sprintf(_$("%d- %s  %d ea"), .@choices[.@i], getitemname(.@choices[.@i+1]), .@choices[.@i+2]);
 			++.@numitemchoices;
 		}
-		.@list$ += .@choices[.@i] + "- " + .@ordinal$ + " Lottery Chance!:" + .@changepage$ +":Cancel";
+		.@list$ +=sprintf(_$("%d- %s Lottery Chance!:%s:Cancel"), .@choices[.@i], .@ordinal$, .@changepage$);
 		next;
 		.@chosen = select(.@list$) - 1;
 		if (.@chosen <= .@numitemchoices) {
 			if (RESRVPTS < .@choices[.@chosen*3]) {
-				mes "[Kafra]";
-				mes "I'm sorry but you do not have enough reserve points for that selection.";
+				mes("[Kafra]");
+				mes("I'm sorry but you do not have enough reserve points for that selection.");
 				continue;
 			}
 			RESRVPTS -= .@choices[.@chosen * 3];
 			if (.@chosen < .@numitemchoices) {
 				// Item
-				mes "[Kafra]";
-				mes "Here you are.";
+				mes("[Kafra]");
+				mes("Here you are.");
 				getitem .@choices[.@chosen*3 + 1], .@choices[.@chosen*3 + 2];
 				close;
 			}
 			// Lottery chance
-			mes "^0000FF"+ .@ordinal$ + " Lottery Opportunity!!^000000";
+			mesf("^0000FF %s Lottery Opportunity!!^000000",.@ordinal$);
```

Comment:
use `F_MesColor` function here
look, if using google machine translation the `^000000` becomes `^ 000000` <-- it adds a space,
using F_MesColor function will eliminate this problem